presto-jdbc/src/test/java/io/prestosql/jdbc/TestPrestoDriverUri.java

```diff
@@ -234,6 +266,39 @@ public void testUriWithSslEnabledPassword()
         assertEquals(properties.getProperty(SSL_TRUST_STORE_PASSWORD.getKey()), "password");
     }
 
+    @Test
+    public void testUriWithSslEnabledUsing443SslVerificationFull()
+            throws SQLException
+    {
+        PrestoDriverUri parameters = createDriverUri("presto://localhost:443/blackhole?SSL=true&SSLVerification=FULL");
+        assertUriPortScheme(parameters, 443, "https");
+
+        Properties properties = parameters.getProperties();
+        assertEquals(properties.getProperty(SSL_VERIFICATION.getKey()), FULL.name());
+    }
+
+    @Test
+    public void testUriWithSslEnabledUsing443SslVerificationCA()
+            throws SQLException
+    {
+        PrestoDriverUri parameters = createDriverUri("presto://localhost:443/blackhole?SSL=true&SSLVerification=CA");
+        assertUriPortScheme(parameters, 443, "https");
+
+        Properties properties = parameters.getProperties();
+        assertEquals(properties.getProperty(SSL_VERIFICATION.getKey()), CA.name());
+    }
+
+    @Test
+    public void testUriWithSslEnabledUsing443SslVerificationNONE()
+            throws SQLException
+    {
+        PrestoDriverUri parameters = createDriverUri("presto://localhost:443/blackhole?SSL=true&SSLVerification=NONE");
+        assertUriPortScheme(parameters, 443, "https");
+
+        Properties properties = parameters.getProperties();
+        assertEquals(properties.getProperty(SSL_VERIFICATION.getKey()), NONE.name());
+    }
+
```

Comment:
Can you please add a test where port 443 is used and when there is no `SSL=true` given but `SSLVerification` is set.

Reply:
Added a test which expects an exception `java.sql.SQLException: Connection property 'SSLVerification' is not allowed`.
This matches the behavior of `SslKeyStorePath` and `SslTrustStorePath`.